### ISSUE TITLE
Unable to use the app with VoiceOver

### ISSUE THREAD

**Issue by AppleUser03** [overtake/TelegramSwift#1011]
The app is completely inaccessible with VoiceOver, so it's impossible for blind users to use the app. I think that Telegram is the best messaging app, but in this way a big part of users can't enjoy it. I hope you will fix this issue as soon as possible. If you want I can test the betas so I can report any problem.
